hw_5_beginning-OOP/main.py

```diff
@@ -0,0 +1,91 @@
+from flask import Flask, jsonify
+from webargs import fields
+from webargs.flaskparser import use_kwargs
+from database_handler import execute_query
+
+
+app = Flask(__name__)
+
+
+@app.route('/popular_genres_by_city')
```

Comment:
[inform]: I changed the name of the route from `/stats_by_city` to `/popular_genres_by_city` because in my opinion we can more clearly indicate what specific statistics our view returns.